lisa/runner.py

```diff
@@ -367,3 +366,15 @@ def _start_loop(self) -> None:
                         # workers are present but no task to run.
                         self._log.debug("Idle worker available but no new runner...")
                         break
+
+    def _cleanup(self) -> None:
+        try:
+            for runner in self._runners:
+                runner.close()
+        except Exception as identifier:
+            self._log.warn(f"error on close runner: {identifier}")
+
+        try:
+            transformer.run(self._runbook_builder, constants.TRANSFORMER_PHASE_CLEANUP)
```

Comment:
[nit] should we add a transformer.close() instead?

Reply:
The `transformer` here is a module name, it's not an object. In this method, it run transformers one by one. Once a transformer is completed, it's discarded.